modules/machineset-creating.adoc

```diff
@@ -33,17 +33,38 @@ endif::[]
 [id="machineset-creating_{context}"]
 = Creating a compute machine set
 
-In addition to the ones created by the installation program, you can create your own compute machine sets to dynamically manage the machine compute resources for specific workloads of your choice.
+In addition to the compute machine sets created by the installation program, you can create your own  to dynamically manage the machine compute resources for specific workloads of your choice.
+
+ifdef::vsphere[]
+[NOTE]
+====
+Clusters that are installed with user-provisioned infrastructure have a different networking stack than clusters with infrastructure that is provisioned by the installation program. As a result of this difference, automatic load balancer management is unsupported on clusters that have user-provisioned infrastructure. Compute machine sets cannot add ingress or DNS compute machines to a cluster with user-provisioned infrastructure.
+====
+endif::vsphere[]
 
 .Prerequisites
 
 * Deploy an {product-title} cluster.
 * Install the OpenShift CLI (`oc`).
 * Log in to `oc` as a user with `cluster-admin` permission.
 ifdef::vsphere[]
-* Create a tag inside your vCenter instance based on the cluster API name. This tag is utilized by the compute machine set to associate the {product-title} nodes to the provisioned virtual machines (VM). For directions on creating tags in vCenter, see the VMware documentation for link:https://docs.vmware.com/en/VMware-vSphere/6.7/com.vmware.vsphere.vcenterhost.doc/GUID-E8E854DD-AA97-4E0C-8419-CE84F93C4058.html[vSphere Tags and Attributes].
+* Create a tag inside your vCenter instance called `<infrastructure_id>`, where `<infrastructure_id>` is the infrastructure ID for the cluster. Assign the tag to a category called `openshift-<infrastructure_id>`.
```

Comment:
For ipi, we don't need to create tag on vcenter
For upi, before 4.6, when creating machine we need to create a tag on vcenter, >=4.6, we don't need to create tag on vcenter.

Reply:
ah, ok thanks! Since we only got as far back as 4.9, I will remove the tag info from docs.